src/local/load.go

```diff
@@ -23,18 +22,25 @@ func (m *Manager) LoadModel(record manifest.Record, target interface{}) (found b
 	}
 
 	// Load values from the meta file
+	metaFilePath := m.Naming().MetaFilePath(record.RelativePath())
 	errPrefix := record.Kind().Name + " metadata"
-	if err := utils.ReadTaggedFields(m.ProjectDir(), record.MetaFilePath(), model.MetaFileTag, errPrefix, target); err != nil {
+	if err := utils.ReadTaggedFields(m.ProjectDir(), metaFilePath, model.MetaFileTag, target, errPrefix); err == nil {
+		record.AddRelatedPath(metaFilePath)
```

Comment:
ked sa nacita `meta.json`, alebo `config.json` tak si cestu poznacime, a nemusime to generovat opat, ked sa riesia `tracked/untracked` paths, vid dalej.